libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/Experimental.kt

```diff
@@ -0,0 +1,130 @@
+package com.mapbox.navigation.dropin
```

Comment:
Moving components to their own packages has little benefit if we still need the drop-in module to initialize them. Proposal (pushed in https://github.com/mapbox/mapbox-navigation-android/tree/lp-tr-audio-component-installation for reference):
_move this to `libnavui-base`_
```kotlin
@ExperimentalPreviewMapboxNavigationAPI
sealed interface ComponentConfig {
    fun component(component: UIComponent)
}

@ExperimentalPreviewMapboxNavigationAPI
internal class NavigationComponents(
    private val components: MapboxNavigationObserverChain = MapboxNavigationObserverChain()
) : ComponentConfig, MapboxNavigationObserver by components {

    override fun component(component: UIComponent) = components.add(component)
}
```

_add this to `libnavui-voice`_
```kotlin
@OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
fun ComponentConfig.audioGuidanceButtonComponent(button: MapboxAudioGuidanceButton) {
    // need to fix shared app flow
    component(AudioGuidanceButtonComponent(button))
}
```

_and add this to `libnavui-maps`_
```kotlin
@OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
fun ComponentConfig.routeLineComponent(
    mapView: MapView,
    config: RouteLineComponentConfig.() -> Unit = {}
) {
    val componentConfig = RouteLineComponentConfig(mapView.context).apply(config)
    component(RouteLineComponent(mapView, componentConfig.options))
}

@OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
fun ComponentConfig.routeArrowComponent(
    mapView: MapView,
    config: RouteArrowComponentConfig.() -> Unit = {}
) {
    val componentConfig = RouteArrowComponentConfig(mapView.context).apply(config)
    component(RouteArrowComponent(mapView, componentConfig.options))
}

class RouteLineComponentConfig(context: Context) {
    var options = MapboxRouteLineOptions.Builder(context)
        .withRouteLineResources(RouteLineResources.Builder().build())
        .withRouteLineBelowLayerId("road-label-navigation")
        .withVanishingRouteLineEnabled(true)
        .build()
}

class RouteArrowComponentConfig internal constructor(context: Context) {
    var options = RouteArrowOptions.Builder(context)
        .withAboveLayerId(RouteLayerConstants.TOP_LEVEL_ROUTE_LINE_LAYER_ID)
        .build()
}
```